sdk/azcore/CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
 
+## 1.8.0 (2023-10-05)
+
+### Features Added
+
+* `messaging/CloudEvent` allows you to serialize/deserialize CloudEvents, as described in the CloudEvents 1.0 specification: [link](https://github.com/cloudevents/spec)
+* Added `TokenRequestOptions.EnableCAE` to indicate whether to request a CAE token
```

Comment:
@chlowell is this the only changelog note needed for CAE?

Reply:
Hmm looks like these notes should also be added.

https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/CHANGELOG.md#features-added-10

Let me know if there are others.

Reply:
Cross-tenant ARM and multitenant auth are already GA, so the new stuff for this release v1.8.0 is:
- Added `Claims` and `EnableCAE` fields to `TokenRequestOptions`
- ARM bearer token policy handles CAE challenges

Reply:
Fixed.